src/Sentry.Unity/NativeUtils/CFunctions.cs

```diff
@@ -148,6 +148,9 @@ internal static void SetValueIfNotNull(sentry_value_t obj, string key, double? v
     [DllImport("sentry")]
     internal static extern void sentry_remove_extra(string key);
 
+    [DllImport("sentry")]
+    internal static extern void sentry_set_trace(string traceId, string parentSpanId);
```

Comment:
question: Do the parameter names match the native function, or could/should we change `parentSpanId` to `spanId` for consistency?

Reply:
Good catch, I'll update the call, thanks!